### DISCUSSION TITLE
neither osu!stable nor osu!lazer works for me. help

### DISCUSSION THREAD

**Discussion by TheN001** [ppy/osu#16424] [Q&A] [answered]
I installed osu! and osu!lazer trough lutris, tried launching them, osu! opened, updated and when the osu! logo appeared it just closed. when i tried launching osu!lazer, it launched but got stuck on a black screen with text at the bottom half way up saying the version of the AppImage. I tried reinstalling them multiple times but it didn't work, i also tried launching them directly without lutris, same thing. I tried launching osu! in bottles, same... i tried updating wine, lutris and bottles, didn't work. idk whats happening with them, nor do i know how to fix it, so i hope you smart ppl can help me fix this.

This is how osu!lazer looks https://i.imgur.com/OfJk9zZ.png

Regular osu! just closes after this https://i.imgur.com/kXzMBYJ.png

Here is my neofetch if it's of any help https://i.imgur.com/vpnmlMF.png

[database.log](https://github.com/ppy/osu/files/7852555/database.log)
[network.log](https://github.com/ppy/osu/files/7852556/network.log)
[performance.log](https://github.com/ppy/osu/files/7852557/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7852558/runtime.log)

**Comment by smoogipoo** [comment]
Please read https://github.com/ppy/osu/discussions/15188

**Reply by TheN001** [reply]
sorry, my bad. totally forgot to upload the logs. i uploaded them now

**Comment by bdach** [comment]
lazer issue is likely https://github.com/ppy/osu-framework/issues/3297

**Comment by TheN001** [comment, marked as answer]
so, i got into the game by changing the intro from triangles to circles. sound wasnt working, i went into settings and tried every single output device there is, some showed [error code 46](https://i.imgur.com/UZRRMR4.png), some got distorted output, but the one that worked was `PulseAudio Sound Server`, i wanted to restart the game for some reason and now the game wont start... i tried deleting the whole folder where the game.ini was, didn't work, tried running the osu.AppImage in terminal, no output at all, i did the `chmod +x osu.AppImage`. 

these were the logs before i deleted the folder where the configs were
[database.log](https://github.com/ppy/osu/files/7859229/database.log)
[network.log](https://github.com/ppy/osu/files/7859230/network.log)
[performance.log](https://github.com/ppy/osu/files/7859231/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7859232/runtime.log)

EDIT: after i rebooted, the game worked
EDIT2: so i changed the audio output from `PulseAudio` to `PipeWire` and the game works fine, changed back to triangles intro and everything works fine, i think the game didnt want to launch cuz i have pipewire installed and not pulseaudio, so something got confused there and didnt want to launch

**Reply by bdach** [reply]
as i suspected you've ran into https://github.com/ppy/osu-framework/issues/3297. good that you've made it work though.
